ods_ci/tests/Tests/0200__rhoai_upgrade/0201__pre_upgrade.robot

```diff
@@ -42,21 +42,21 @@ ${UPGRADE_CONFIG_MAP}    upgrade-config-map
 *** Test Cases ***
 Set PVC Size Via UI
     [Documentation]    Sets a Pod toleration via the admin UI
-    [Tags]      Upgrade
+    [Tags]      Upgrade    Dashboard
     [Setup]     Begin Web Test
     Set PVC Value In RHODS Dashboard        ${S_SIZE}
     [Teardown]      Dashboard Test Teardown
 
 Set Culler Timeout
     [Documentation]     Sets a culler timeout via the admin UI
-    [Tags]      Upgrade
+    [Tags]      Upgrade    IDE
```

Comment:
```suggestion
    [Tags]      Upgrade    Dashboard
```

This only touches Dashboard, and dashboard creates a configmap. notebook-controller is not involved during this step, so test failure here is unlikely to be fixed by IDE team.

Reply:
Updated!!